tracpro/templates/frame.html

```diff
@@ -232,28 +235,58 @@
                           <span class='caret'></span>
                         </a>
                         <ul class='dropdown-menu' role='menu'>
-                          {% for region in request.user_regions %}
-                            <li>
-                              <a href='{{ request.path }}?_region={{ region.pk }}'>
-                                {{ region.name }}
-                              </a>
-                            </li>
-                          {% endfor %}
                           {% if user_is_admin %}
-                            <li class='divider'></li>
                             <li>
                               <a href='{{ request.path }}?_region=0'>
                                 {% trans "All Regions" %}
                               </a>
                             </li>
                           {% endif %}
+                          {% if show_subregions_toggle_form %}
+                            <li>
+                              <form id="toggle-subregions-form"
+                                    method="POST" action="{% url "toggle-subregions" %}">
+                                {% csrf_token %}
+                                <input type="hidden" name="include_subregions"
+                                       value="{% if request.include_subregions %}0{% else %}1{% endif %}" />
+                                <input type="hidden" name="next"
+                                       value="{{ request.get_full_path }}" />
+                                <input type="submit" class="hidden" />
+                              </form>
+                              {% if request.include_subregions %}
+                                <a class="toggle-subregions" href="#"
+                                   title="Currently showing data for {{ request.region }} and all sub-regions. Click to show data from {{ request.region }} only.">
```

Comment:
Really like the title for this and the Show link, they give the user great feedback.